features/admin.sms-providers.v1/pages/vonage-sms-provider.tsx

```diff
@@ -43,6 +44,24 @@ const VonageSMSProvider: FunctionComponent<VonageSMSProviderPageInterface> = (
     } = props;
 
     const { t } = useTranslation();
+    const [ isShow, setIsShow ] = useState(false);
+
+    const renderInputAdornment = (): ReactElement => {
+
+        return (
+            <InputAdornment position="end">
+                <Icon
```

Comment:
can't we use Oxygen icons here? to make it consistent with our password Textfields

Reply:
Since we need to have an `onclick` action and the `icon` needs to be changed dynamically, we can use the semantic icons here.